### PR TITLE
Add a starting point for Muriel Typography

### PR DESCRIPTION
**Pull request by blowery**
#### Changes proposed in this Pull Request

* Adds a place to put rules / mixins for muriel-based typography rules

### CODE REVIEW

File: packages/muriel-typography/typography.scss [hunk 1]
```diff
@@ -0,0 +1,70 @@
+.muriel-text-display-level-1 {
+  font-weight: regular;
```
Comment (drw158) [inline packages/muriel-typography/typography.scss]:
```suggestion
  font-weight: 400;
```

Same for all instances of font-weight: ~regular~ normal;

**Review by davewhitley — CHANGES_REQUESTED** [review]
Looks like a good start aside from my comment.

**Comment by blowery** [comment]
@drw158 what do you think of the names? Do we want mixins in additions to the utility classes?

**Comment by drw158** [comment]
I just cleaned up the CSS and renamed Overline

https://codepen.io/drw158/pen/zbgEqO

```css
.display-large {
  font-weight: 400;
  font-size: 32px;
}

.display-medium {
  font-weight: 400;
  font-size: 24px;
}

.display-small {
  font-weight: 400;
  font-size: 20px;
}

.subtitle {
  font-weight: 600;
  font-size: 16px;
}

.subtitle-small {
  font-weight: 600;
  font-size: 14px;
}

.body {
  font-weight: 400;
  font-size: 16px;
  line-height: 22px;
}

.body-small {
  font-weight: 400;
  font-size: 14px;
  line-height: 20px;
}

.button {
  font-weight: 600;
  font-size: 14px;
}

.caption {
  font-weight: 400;
  font-size: 12px;
  line-height: 16px;
}

.title-small {
  font-weight: 600;
  font-size: 12px;
  text-transform: uppercase;
  letter-spacing: 1px;
  line-height: 16px;
}```

**Comment by drw158** [comment]
I just went ahead and made the changes. I don't think we should do mixins because once ppl start using them, it'll be hard to remove them if we decide they are a problem.

I don't know how I feel about the muriel prefix. If I'm using the class names, I'd rather just write "text-..." or for the css variables, "--color-...". It's less verbose.

**Comment by blowery** [comment]
LGTM. Want to land it as is and move on?

**Review by davewhitley — APPROVED** [review]
👍

**Comment by scruffian** [comment]
>  I don't think we should do mixins yet because once ppl start using them, it'll be hard to remove them if we decide they are a problem.

@drw158 could you explain your reasoning here? Why would mixins be a problem?

**Comment by drw158** [comment]
Large-scale CSS usage is not my forte, but I think the common argument is that it will introduce a lot of repetition in the CSS. But, I guess that's not a problem with gzip files?

https://csswizardry.com/2016/02/mixins-better-for-performance/

Mostly, I wanted to hold off on mixins until some more framework minded folks could weigh in before we implement something.

**Comment by scruffian** [comment]
My concern is that we start using these class names in our CSS instead of following the naming conventions we already have. Maybe that's actually more efficient, but if that's the approach we are taking we should be deliberate about it...

**Pull request merged**